testing/cloudbuild/run.bash

```diff
@@ -20,6 +20,8 @@ trap "gimmeproj -project golang-samples-tests done $GOLANG_SAMPLES_PROJECT_ID" E
 set -x
 
 export GOLANG_SAMPLES_SPANNER=projects/golang-samples-tests/instances/golang-samples-tests
+export GOLANG_SAMPLES_BIGTABLE_PROJECT=golang-samples-tests
+export GOLANG_SAMPLES_BIGTABLE_INSTANCE=testing-instance
```

Comment:
Sorry, this file ins't really used. These updates should go in https://github.com/GoogleCloudPlatform/golang-samples/blob/master/testing/kokoro/system_tests.sh.

(I'm guessing the new test is being skipped right now.)